.github/actions/tp-integration-tests/cdn.json

```diff
@@ -13,11 +13,11 @@
 	"traffic_ops_golang": {
 		"insecure": true,
 		"port": "6443",
-		"log_location_error": "error.log",
-		"log_location_warning": "warning.log",
-		"log_location_info": "info.log",
-		"log_location_debug": null,
-		"log_location_event": "event.log",
+		"log_location_error": "stderr",
+		"log_location_warning": "stderr",
+		"log_location_info": "stdout",
+		"log_location_debug": "stderr",
```

Comment:
Shouldn't this be `stdout`?

Reply:
It is

Reply:
or did you mean like "should this _really_ be stdout instead of stderr?" Or were you talking about "log_location_debug"? There was a point when "log_location_info" was set to just "out" instead of "stdout", so I assumed you made this comment before I fixed that typo, but now that I think about it I'm not sure.

Reply:
I meant since info is stdout shouldn't debug also be stdout

Reply:
eh, sure. It doesn't make much difference, honestly maybe they should all just be the same stream. Idk, I'll just make it stdout for now.